Schemas/guid.xsd

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- With XMLSpy v2011 rel. 2 sp1 (http://www.altova.com) created by Jon Anders Sollien (Catenda AS) -->
```

Comment:
Should we remove these comments from all of our xsd's? I vote yes

Reply:
Yes please